src/theme/components.js

```diff
@@ -249,6 +249,12 @@ const components = {
           outline: '1px auto',
         },
       },
+      select: {
+        '&:focus': {
+          backgroundColor: Colors.Background300Dark,
+          borderRadius: '4px',
```

Comment:
> This borderRadius style is not necessary; it is already taken care of in `src/styles/DropdownMenu.ts` Speaking of that file, I noticed the wrong color bug happens because of this. line
> 
> https://github.com/Netacci/perfcompare/blob/6aa592eac274eef1fb470b86144aace635560fb3/src/styles/DropDownMenu.ts#L68.
> 
> You can go ahead and delete the focus styles for both dark and light in this file because we will handle them below in the component file instead.

Oh alright. I missed that file during my troubleshooting :)